.github/workflows/content.yml

```diff
@@ -0,0 +1,152 @@
+name: Release Academy Content
```

Comment:
This file should not be here, please check the current workflow in master and remove if not needed.